src/lib.rs

```diff
@@ -87,6 +87,7 @@ impl Aviffy {
         let alpha_image_id = 2;
         let high_bitdepth = depth_bits >= 10;
         let twelve_bit = depth_bits >= 12;
+        const ESSENTIAL_BIT: u8 = 0x80;
```

Comment:
Here is the relevant definition from HEIF (ISO/IEC 23008-12:2017) Section 9.3.2 (page 29):

```
aligned(8) class ItemPropertyAssociation
    extends FullBox('ipma', version, flags)
{
    unsigned int(32) entry_count;
    for(i = 0; i < entry_count; i++) {
        if (version < 1)
          unsigned int(16) item_ID;
        else
          unsigned int(32) item_ID;
        unsigned int(8) association_count;
        for (i=0; i<association_count; i++) {
           bit(1) essential;
           if (flags & 1)
             unsigned int(15) property_index;
           else
             unsigned int(7) property_index;
        }
    }
}
```